its/it/azdo.ts

```diff
@@ -0,0 +1,39 @@
+import * as vm from "azure-devops-node-api";
+import { Build, BuildStatus } from "azure-devops-node-api/interfaces/BuildInterfaces";
+import { AZDO_BASE_URL, AZDO_ORGANIZATION, AZDO_PROJECT } from "../constant";
+import { loadEnvironmentVariables } from "./env";
+
+export function getAzdoApi(): vm.WebApi {
+  const env = loadEnvironmentVariables();
+  const credentialHandler = vm.getPersonalAccessTokenHandler(env.AZURE_TOKEN);
+  return new vm.WebApi(AZDO_BASE_URL + AZDO_ORGANIZATION, credentialHandler);
```

Comment:
you'll have to show me what is required to setup / add / remove additional configurations

Reply:
Yes, after this PR, we still need to manually click on AzDO to register the GitHub pipelines there, but the goal of the next task is to automate the pipeline creation with the Web API

Reply:
It makes sense now, I realize the AZURE_TOKEN is the PAT you setup and map in the library of the project. very cool they have a node API that's actually easy to work with (wasn't expecting that 😅 )